web_src/js/index.js

```diff
@@ -2491,6 +2490,7 @@ $(document).ready(async () => {
     initClipboard(),
     initUserHeatmap(),
     initServiceWorker(),
+    initNotificationCount(),
```

Comment:
This change is not strictly necessary but as it is technically async this seems reasonable to do so.